### PR TITLE
DO NOT MERGE LPS-100139 - Migrate AssetTagsSelector to react

### PR DESCRIPTION
**Pull request by carloslancha**

### CODE REVIEW

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java [hunk 1]
```diff
@@ -52,6 +51,42 @@
...
+
+	public String getHiddenInput() {
+		return _hiddenInput;
+	}
+
+	public String getRemoveCallback() {
+		return _removeCallback;
+	}
+
+	public boolean isAllowAddEntry() {
```
Comment (adolfopa) [inline modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java]: Methods need to be alphabetically sorted (run ant format-source from portal-impl/ directory).

**Comment by carloslancha** [comment]
ci:test

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java [hunk 2]
```diff
@@ -217,46 +257,35 @@ protected String getTagNames() {
...
+		data.put("inputName", _getInputName());
+		data.put("portletURL", getPortletURL().toString());
 
-			selectedItems.add(item);
+		if (Validator.isNotNull(_addCallback)) {
+			data.put("removeCallback", _getNamespace() + _removeCallback);
 		}
 
-		context.put("selectedItems", selectedItems);
+		List<String> selectedItems = StringUtil.split(getTagNames());
```
Comment (adolfopa) [inline modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java]: You can inline this if there are no more usages.

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/page.jsp [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+--%>
+
+<%@ include file="/asset_tags_selector/init.jsp" %>
+
+<%
+Map<String, Object> data = (Map<String, Object>)request.getAttribute("liferay-asset:asset-tags-selector:data");
```
Comment (adolfopa) [inline modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/page.jsp]: Sort alphabetically.

File: modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java (hunk 1 above)
Comment (carloslancha) [reply modules/apps/asset/asset-taglib/src/main/java/com/liferay/asset/taglib/servlet/taglib/AssetTagsSelectorTag.java]: They're sorted, aren't them?

File: modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js [hunk 4]
```diff
@@ -12,118 +12,104 @@
...
+						start: 0,
+						tagProperties: ''
+					}
+				}),
+				p_auth: Liferay.authToken
+			}),
+			method: 'POST'
+		},
+		link:
+			window.location.origin +
```
Comment (boton) [inline modules/apps/asset/asset-taglib/src/main/resources/META-INF/resources/asset_tags_selector/AssetTagsSelector.es.js]: Maybe use template literal

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by carloslancha** [comment]
ci:test:sf

**Comment by julien** [comment]
Just started reviewing :)

**Pull request closed without merging**